lib/ansible/plugins/httpapi/fortios.py

```diff
@@ -46,6 +46,7 @@
 import urllib
```

Comment:
```suggestion
from ansible.module_utils.six.moves import urllib
```

If you do this, you don't have to make any of the changes below, and it will find the correct urllib quote for the running version